.travis.yml

```diff
@@ -37,6 +37,7 @@ jobs:
       - make test-e2e
       - echo -e '\n\n======= manager logs =======\n\n' && kubectl logs -n gatekeeper-system gatekeeper-controller-manager-0 manager
     - stage: "build, test, deploy"
+      if: branch = master AND repo = open-policy-agent/gatekeeper AND type != pull_request
```

Comment:
Why is this being added?

Reply:
This ensures that this stage does not run on a PR, only on a commit to master branch

Reply:
We want this stage to be run in  PRs, as it's where unit tests are run. Whether a build is pushed to Quay is determined by the `deploy` subfield, which only runs on the master branch.

Reply:
Ok I see the issue. This stage includes `make` which runs the unit tests AND `make travis-dev-deploy`. So for `make travis-dev-deploy` we do not want it to run on a PR, but for `make` we do want it to run on each PR. Let me fix this as well.

Reply:
+1 we need this to run on PRs, this includes UTs